qiskit/scheduler/methods/basic.py

```diff
@@ -157,16 +157,15 @@ def get_measure_schedule() -> CircuitPulseDef:
             default_sched = inst_map.get('measure', qubits)
             for time, inst in default_sched.instructions:
                 if isinstance(inst, AcquireInstruction):
-                    mem_slots = []
                     for channel in inst.acquires:
```

Comment:
`acquires` is deprecated now, so this causes a ton of deprecation warnings to be printed. We can either wait to deprecate `acquires`, use the private attribute (probably not the best idea), or suppress warnings around this line of code. The first might be the simplest and easiest to do.